models/ContainerPage.php

```diff
@@ -67,10 +69,84 @@ public function rules()
     {
         $rules = $this->defaultRules();
         $rules[] = ['in_new_window', 'integer'];
-        $rules[] = [['page_content'], 'safe'];
+	    $rules[] = [['page_content'], 'safe', 'when' => array($this, 'isNotHtml')];
+	    $rules[] = [['page_content'], 'filter', 'when' => array($this, 'isHtml'), 'filter' => array($this, 'purifyFilter')];
+
```

Comment:
I think we need the same for title? or is any content by default santized/html escaped?

Reply:
I think it is sanitized by default but I need to check it later

Reply:
added a stripping logic for the title